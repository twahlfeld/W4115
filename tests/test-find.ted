int main() {
  Page p = get("http://haighr.com");
  Element e = p.find("class", "boxContainer");
  Element f = e.find("type", "h4");
  Element g = p.find("id", "secondBox");
  Element h = g.find("type", h2"); 
  print(stdout, "%s\n%s\n", f.text, h.text);
  return 0;
}
