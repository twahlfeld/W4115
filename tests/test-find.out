Mishki, Inc.
Partner/Co-Founder
